SHARDING.md

```diff
@@ -0,0 +1,254 @@
+
+Update the TrustedRoot for the new shard's key material:
+
+```
+{
+  "mediaType": "...",
+  "tlogs": [
+    // new shard
+    {
+      "baseUrl": "https://<year-revions>.rekor.(sigstore|sigstage).dev",
+      "hashAlgorithm": "SHA2_256",
+      "publicKey": {
+        "rawBytes": "base64-encoded DER public key",
+        "keyDetails": "ED25199",
+        "validFor": {
+          "start": "<UTC timestamp for when shard was spun up>"
+        }
+      },
+      "logId": {
+        "keyId": "wNI9atQGlz+VWfO6LRygH4QUfY/8W4RFwiT5i5WRgB0="
+      }
+    },
+    // previous shard
+    {
+      "baseUrl": "...",
+      "hashAlgorithm": "SHA2_256",
+      "publicKey": {
+        "rawBytes": "...",
+        "keyDetails": "...",
+        "validFor": {
+          "start": "...",
+          "end": "<UTC timestamp for approx when the shard will be shut down>"
+        }
+      },
+      "logId": {
+        "keyId": "..."
+      }
+    },
+    ...
+  ],
+  ...
+}
+```
+
+
+Update the SigningConfig for the new shard:
+
+```
+{
+    "media_type": "...",
+    "ca_urls": [...],
+    "oidc_urls": [...],
+    // sorted from newest to oldest
+    "rekor_tlog_urls": [
+        // new shard
+        {
+            "url": "<year-revision>.rekor.(sigstore|sigstage).dev",
+            "major_api_version": 2,
+             "valid_for": {
+                "start": "<UTC timestamp, at least one week past when to-be-signed TUF timestamp will expire>",
+            }
+        },
+        // previous shard
+        {
+            ...,
+            "valid_for": {
+                "start": "...",
+                "end": "<valid_for.start for new shard>"
+            }
+        }
+    ],
+    "rekor_tlog_config": {...},
+    "tsa_urls": [...],
+    "tsa_config": {...},
+}
+```
+
+Follow the playbooks in https://github.com/sigstore/root-signing and
+https://github.com/sigstore/root-signing-staging to orchestrate a new
+signing event.
+
+## Turn Down Old Shard
+
+Wait a week, until all clients have received updated SigningConfigs
+and started signing using the new shard, and are using the updated TrustedRoot
+to verify inclusion proofs.
+
+Verify write traffic to the previous shard has gone down to ~0 QPS.
+
+<!--- TODO: Update with link to monitoring -->
+
+### Prober
+
+Update the prober configuration to remove the previous shard URL for testing writes.
+
+### K8s
+
+Turn down the previous shard by removing the previous values for `rekor-tiles.shards`.
+We can turn down the Kubernetes pods because the server only serves the write path -
+the read path is exclusively served via GCP resources (Load Balancer, Storage).
+
+Merge and wait for ArgoCD to shut down the server.
+
+### Remove GCP Resources
+
+We only need a subset of GCP resources to serve read traffic. To save costs,
+we can turn down the database and destroy the signing key.
+
+<!--- TODO: Specify how we should do this -->
+
+<!--- TODO: Specify how to convey to monitors the shard is frozen.
```

Comment:
One other thought - For CT, we can't add extension lines, so we if we want to denote a frozen shard in the same way, baking in the checkpoint might be the only viable approach.